lock-api/src/main/java/com/palantir/lock/client/RemoteTimelockServiceAdapter.java

```diff
@@ -121,6 +130,28 @@ public static RemoteTimelockServiceAdapter create(
                 minLeasedTimestampGetter);
     }
 
+    public static RemoteTimelockServiceAdapter create(
```

Comment:
To keep in mind for the weirdness in this class https://github.com/palantir/atlasdb/pull/7398#discussion_r1824772592

As I said there, the public constructor + public factory situation existed before and I can track standardizing in a separate PR if you'd like that

Reply:
Ah, okay. Womp - wasn't aware of that - let's do that if feasible? Thanks!

Reply:
Tracking internally!